packages/app/src/components/drawer/accounts/switch-accounts.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Box, Flex, Text, Button } from '@stacks/ui';
+import { useWallet } from '@common/hooks/use-wallet';
+import { CheckmarkIcon } from '@components/icons/checkmark-icon';
+import { useRecoilValue, useSetRecoilState } from 'recoil';
+import { accountDrawerStep, AccountStep } from '@store/recoil/drawers';
+import { getAccountDisplayName, getStxAddress } from '@stacks/wallet-sdk';
+import { currentTransactionVersion } from '@store/recoil/networks';
+
+interface SwitchAccountProps {
+  close: () => void;
+}
+export const SwitchAccounts: React.FC<SwitchAccountProps> = ({ close }) => {
+  const { wallet, currentAccountIndex, doSwitchAccount } = useWallet();
+  const setAccountDrawerStep = useSetRecoilState(accountDrawerStep);
+  const transactionVersion = useRecoilValue(currentTransactionVersion);
+  const accountRows = (wallet?.accounts || []).map((account, index) => {
+    return (
+      <Box
+        width="100%"
+        key={`account-${account.index}`}
+        _hover={{
+          letterSpacing: '0.25px',
+        }}
+        transition="100ms letter-spacing"
```

Comment:
🤔

Reply:
lol we discussed this hover state like 3 months ago and no one had a better idea 🙃